.github/workflows/release-build-all.yml

```diff
@@ -0,0 +1,199 @@
+name: Release - Build All
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, closed]
+    branches:
+      - main
+      - "release/**"
+      - "hotfix/**"
+  workflow_dispatch:
+
+env:
+  BUILD_NAMESPACE: 0c27fb-tools
```

Comment:
Not a part of this PR. Namespace could be a non-secret env from github?

Reply:
I like the idea, I will do it in an upcoming PR.